python/django_bolt/auth/user_loader.py

```
@@ -31,17 +51,88 @@ def _get_executor() -> concurrent.futures.ThreadPoolExecutor:
+            # the executor, regardless of handler context.
+            def run_async_get_user():
+                return asyncio.run(get_user(user_id, auth_context or {}))
+
+            return _get_executor().submit(run_async_get_user).result()
+
+        return load_via_async
+
+    return None
+
+
+def default_django_user_loader(user_id: str, auth_context: dict | None, is_async_context: bool) -> Any:
```

Ruff (0.15.20)

[warning] Unused function argument: `auth_context`

(ARG001)